README.md

```diff
@@ -1,2 +1,5 @@
-# Trial_Review
-A review on one of my Trial Project
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
```

Comment:
Please, add a better description of your project here. This will help the readers have a better understanding of the project.